### PR TITLE
Fixes #19940: add support for ubuntu in parent ticket

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/19940

### CODE REVIEW

**Comment by ncharles** [comment]
PR updated with a new commit

File: contrib/inventory-hooks/security-upgrades.sh [hunk 1]
```diff
@@ -10,6 +10,9 @@ result () {
   exit 0
 }
 
+# Source os-release
+. /etc/os-release
+
 if [ ! -f /etc/debian_version ]; then
```
Comment (amousset) [inline contrib/inventory-hooks/security-upgrades.sh:16]: we can remove this and only rely on /etc/os-release

**Comment by ncharles** [comment]
PR updated with a new commit

File: contrib/inventory-hooks/security-upgrades.sh [hunk 2]
```diff
@@ -10,14 +10,30 @@ result () {
   exit 0
 }
 
-if [ ! -f /etc/debian_version ]; then
-  result -2
+# Source os-release
+. /etc/os-release
+
+if [ "debian" != "$ID" ]; then
```
Comment (amousset) [inline contrib/inventory-hooks/security-upgrades.sh:16]: no need for this with the last line imo

Comment (ncharles) [reply contrib/inventory-hooks/security-upgrades.sh:16]: without it, it will always try the apt-get, and return -1 on non debian system

Comment (amousset) [reply contrib/inventory-hooks/security-upgrades.sh:16]: indeed

**Review by amousset — APPROVED** [review]

**Pull request merged**